fdbclient/StorageServerInterface.h

```diff
@@ -310,6 +318,43 @@ struct GetKeyValuesRequest : TimedRequest {
 	}
 };
 
+struct GetKeyValuesAndFlatMapReply : public LoadBalancedReply {
+	constexpr static FileIdentifier file_identifier = 1783067;
+	Arena arena;
+	VectorRef<KeyValueRef, VecSerStrategy::String> data;
+	Version version; // useful when latestVersion was requested
+	bool more;
+	bool cached = false;
+
+	GetKeyValuesAndFlatMapReply() : version(invalidVersion), more(false), cached(false) {}
+
+	template <class Ar>
+	void serialize(Ar& ar) {
+		serializer(ar, LoadBalancedReply::penalty, LoadBalancedReply::error, data, version, more, cached, arena);
+	}
+};
+
+struct GetKeyValuesAndFlatMapRequest : TimedRequest {
```

Comment:
I think it's worth noting that if one changes GetKeyValuesAndFlatMapRequest/Reply, it also needs to change GetKeyValuesRequest/Reply